service/tasks/driver.py

```diff
@@ -907,6 +908,9 @@ def _deploy_init_to(driverCls, provider, identity, instance_id,
         playbooks = ansible_deploy_to(instance.ip, username, instance_id)
         _update_status_log(instance, "Ansible Finished for %s." % instance.ip)
         logger.debug("_deploy_init_to task finished at %s." % datetime.now())
+    except AnsibleDeployException as exc:
+        logger.exception(exc)
+        _deploy_init_to.retry(exc=exc)
```

Comment:
After talking to @amercer1 it seems like 'when Ansible fails there is usually a reason' and continued retries may be futile... Perhaps we should consider raising-out here?